kong/runloop/plugin_servers/init.lua

```diff
@@ -310,22 +322,22 @@ end
 
 
 function plugin_servers.start()
-  if ngx.worker.id() ~= 0 then
-    kong.log.notice("only worker #0 can manage")
-    return
-  end
-
   local pluginserver_timer = proc_mgmt.pluginserver_timer
+  local pluginserver_connection_check_timer = proc_mgmt.connection_check_timer
+  local native_timer_at = _G.native_timer_at
 
   for _, server_def in ipairs(proc_mgmt.get_server_defs()) do
     if server_def.start_command then
-      _G.native_timer_at(0, pluginserver_timer, server_def)
+      if worker_id() == 0 then
```

Comment:
We can check `worker_id()` before the loop, then use `local call_back = pluginserver_xxx`,
so we could remove `if-end` in the loop.

Reply:
notice that that's not an if-else, the later handler will always run.

Reply:
I saw the code had changed, so I think it should run only one handler in timer, right?